neuraxle/hyperparams/distributions.py

```diff
@@ -487,15 +495,15 @@ def narrow_space_from_best_guess(self, best_guess, kept_space_ratio: float = 0.5
 class RandInt(HyperparameterDistribution):
     """Get a random integer within a range"""
 
-    def __init__(self, min_included: int, max_included: int):
+    def __init__(self, min_included: int, max_included: int, null_default_value=0):
```

Comment:
This is different than `Uniform`. It should maybe be `=None` here too with an if that takes the min. 

Or maybe in all cases, perhaps even the if could take a default value between min and max.